CMakeLists.txt

```diff
@@ -96,11 +96,26 @@ if (OTBR_BACKBONE_ROUTER)
     target_compile_definitions(otbr-config INTERFACE
             OTBR_ENABLE_BACKBONE_ROUTER=1
     )
+    set(OTBR_BACKBONE_ROUTER_DUA_NDPROXYING ON CACHE BOOL "Backbone Router enables DUA ND Proxying by default")
     set(OT_THREAD_VERSION 1.2 CACHE STRING "Backbone Router requires Thread 1.2 or higher" FORCE)
     set(OT_BACKBONE_ROUTER ON CACHE BOOL "Enable Backbone Router feature in OpenThread" FORCE)
     set(OT_SERVICE ON CACHE BOOL "Backbone Router requires Thread network service" FORCE)
 endif()
 
+
+option(OTBR_BACKBONE_ROUTER_DUA_NDPROXYING "Enable Backbone Router DUA ND Proxying" OFF)
+
+if (OTBR_BACKBONE_ROUTER_DUA_NDPROXYING)
+    target_compile_definitions(otbr-config INTERFACE
+            OTBR_ENABLE_BACKBONE_ROUTER_DUA_NDPROXYING=1
+    )
+    set(OT_BACKBONE_ROUTER_DUA_NDPROXYING ON CACHE BOOL "Disable Backbone Router DUA ND Proxying in OpenThread" FORCE)
```

Comment:
```suggestion
    set(OT_BACKBONE_ROUTER_DUA_NDPROXYING OFF CACHE BOOL "Disable Backbone Router DUA ND Proxying in OpenThread" FORCE)
```

Looks like a typo?

Reply:
It's not. If `OTBR_BACKBONE_ROUTER_DUA_NDPROXYING` is ON, we are tuning `OT_BACKBONE_ROUTER_DUA_NDPROXYING` too by **FORCE**.

Reply:
But why does the description say "disable"?

Reply:
fixed